app/controllers/data_resources_controller.rb

```diff
@@ -3,6 +3,6 @@
 class DataResourcesController < ApplicationController
   def index
     @pagy, @data_resources = pagy(DataResources::Fetcher.call(query: params[:query], filters: params[:filters]))
-    @organisations = Organisation.where.not(name: nil).id_name_slug_order_asc
+    @organisations = Organisation.joins(:published_resources).distinct.id_name_slug_order_asc
```

Comment:
Do we still need the slug in the scope, or can we get rid of it?

Reply:
We can get rid now thanks to titleize 🙌 will re-commit